Algorithm.Python/readme.md

```diff
@@ -53,6 +54,7 @@ $ sudo ln -s /<path to anaconda>/lib/libpython* /usr/local/lib
 "algorithm-location": "../../../Algorithm.Python/BasicTemplateAlgorithm.py",
 ```
  3. Rebuild LEAN. This step will ensure that the `Python.Runtime.dll` you set in 2.1 will be used.
+      Note: Rebuilding will clean and build the whole solution from scratch, ignoring anything it's done before.
```

Comment:
"Note: You should do a complete/clean build to recompile the whole solution from scratch, ignoring anything it's done before."

Reply:
would recommend keeping verbiage 'rebuild' somewhere in here since that's the button users need to click in VS